spec/unused_options_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe Fastlane do
+  describe Fastlane::Action do
+    describe "No unused options" do
+      let (:al_exceptions) { %w(pilot appstore cert deliver gym match pem produce scan sigh snapshot supply testflight mailgun testfairy ipa import_from_git hockey deploygate crashlytics artifactory appledoc) }
+
+      Fastlane::ActionsList.all_actions do |action, name|
+        next unless action.available_options.kind_of?(Array)
+        next unless action.available_options.last.kind_of?(FastlaneCore::ConfigItem)
```

Comment:
Are there actions that have options that aren't `ConfigItem`s? Why not check that they're all `ConfigItem`?